Lesson-6-Task-6/src/main/java/Lesson6/Task6/Main.java

```diff
@@ -0,0 +1,10 @@
+package Lesson6.Task6;
+public class Main {
+    public static void main(String[] args){
```

Comment:
Żeby wykonać jedną logikę (zmienić litery na wielkie) trzeba utworzyć dwa obiekty i wywołać trzy metody (i to w odpowiedniej kolejności). To trzeba uprościć

Reply:
Uprościłem :)